sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Metrics/Metrics.cs

```diff
@@ -83,7 +83,7 @@ public override int GetCognitiveComplexity(SyntaxNode node)
             return 0; // Not implemented
         }
 
-        public override int ExecutableLinesCount => 0; // Not implemented
+        public override ICollection<int> ExecutableLines => new List<int>(); // Not implemented

```

Comment:
Confirmed this is fine. When no executable lines are provided, the behaviour would be as it is currently.